### PR TITLE
Implement UI for SignIn Feature

### PR DESCRIPTION
**Pull request by leanhduy**

### CODE REVIEW

File: client/src/components/navbar.js [hunk 1]
```diff
@@ -68,7 +64,6 @@ const Navbar = ({ children, openNewPostDialog }) => {
                         <FavoriteBorderRoundedIcon />
                     </IconButton>
                 </HomeLink>
-                {/* If user is not logged in, redirect to login page. Otherwise, redirect to profile page */}
                 {user ? (
```
Comment (pybotlad) [inline client/src/components/navbar.js:67]: Update this render condition to user `userId` to be consistent with the condition used in `home.js`.

File: server/src/schema.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ const {
 const { DateTimeResolver } = require('graphql-scalars')
 const { getJWT, getHash } = require('./utils/helpers')
 const { compareSync } = require('bcryptjs')
-const { isCompositeType } = require('graphql')
+require('graphql')
```
Comment (pybotlad) [inline server/src/schema.js:14]: Possible unused import, if it doesn't affect the current code, remove it.

**Review by pybotlad — CHANGES_REQUESTED** [review]
Please look at the comments and update the code accordingly. Thanks

File: server/src/schema.js (hunk 2 above)
Comment (leanhduy) [reply server/src/schema.js:14]: Removed per suggestion.

File: client/src/components/navbar.js (hunk 1 above)
Comment (leanhduy) [reply client/src/components/navbar.js:67]: Updated per suggestion.

**Review by pybotlad — APPROVED** [review]
I'm happy with the changes. Please go ahead and merge into `main` branch

**Pull request merged**